pkg/metrics/producers/scheduledcapacity/producer.go

```diff
@@ -25,5 +28,67 @@ type Producer struct {
 
 // Reconcile of the metrics
 func (p *Producer) Reconcile() error {
+	var (
+		loc     *time.Location
+		err     error
+		currVal int32
+	)
+	matchFound := false
+	now := time.Now()
+
+	if p.Spec.Schedule.Timezone != nil {
+		loc, err = time.LoadLocation(*p.Spec.Schedule.Timezone)
+		if err != nil {
+			return fmt.Errorf("timezone was not a valid input")
+		}
+	} else {
+		// Set Default location to UTC if not specified
+		loc = time.UTC
+	}
+	now = now.In(loc)
+
+F:
```

Comment:
Is this necessary?

Reply:
once I get rid of the switch statement, it shouldn't be. Thanks for the tips.